seong-geun/app.js

```diff
@@ -73,6 +73,20 @@ app.post("/posts", async (req, res) => {
   res.status(201).json({ message: "postCreated" });
 });
 
+// 좋아요 누르기
+app.post("/likes", async (req, res) => {
+  const { userId, postId } = req.body;
+  await appDataSource.query(
+    `INSERT INTO likes(
+      user_id,
+      post_id
+    ) VALUES (?, ?);
+    `,
+    [userId, postId]
+  );
+  res.status(200).json({ message: "likeCreated" });
```

Comment:
```suggestion
  return res.status(201).json({ message: "likeCreated" });
```
조금 더 적절한 상태 코드를 활용해주세요.